src/Schema/Events/index.ts

```diff
@@ -22,6 +22,9 @@ import {
   ClickedLoadMore,
   ClickedMainArtworkGrid,
   ClickedNavigationTab,
+  ClickedOnFrameMeasurements,
```

Comment:
The framed measurements portion of the form consists of three input fields. Does "clicked on" capture exactly what we want to track? Do we care which field they focused? Does tabbing to a field mean the same thing as clicking on it? Do we care if they entered values?

Reply:
Thanks Brenda! 

It sounds like we want to fire an event when they focus on each field (height, width, depth, or diameter), so that it fires when they click or tab into the field. I don't see any mechanism in this method to distinguish between _which_ field they focused on -- is that okay?

Reply:
Thank you! Yes, this is ok! We want to avoid such level of granularity in Cohesion; we can still see bottleneck areas or sense patterns by analysing these interactions together with the data we can get from redshift in regards to these fields.